src/app/containers/InlineLink/__snapshots__/index.test.jsx.snap

```diff
@@ -21,7 +21,7 @@ exports[`InlineLinkContainer external link accessibility should be explicitly ma
 
 <div>
   <a
-    aria-label="این لینک هست، لینک خارجی"
+    aria-label="این لینک هست، خارجی"
```

Comment:
Does this have the comma before 'external'? I see it in some translation files, may of been better to add it to the code rather than each translation file though if that's how we've done it in other places then lets follow suit.